drivers/SmartThings/matter-switch/profiles/light-color-level-nl.yml

```diff
@@ -0,0 +1,25 @@
+name: light-color-level-nl
```

Comment:
Change the name to `light-color-level-illuminance-motion`

Update the filename and the `deviceProfileName` in the fingerprint to match the new name